src/SectionTitle.jsx

```diff
@@ -34,6 +33,12 @@ class SectionTitle extends React.Component {
 					</Chunk>
 				</FlexItem>
 				{children &&
+					<Chunk>
+						<FlexItem shrink
+							>
+							{children}
+						</FlexItem>
+					</Chunk>
 					<FlexItem shrink>
 						{children}
```

Comment:
This must have been result of a bad merge or something. I'm only seeing one `<FlexItem>` containing `chlidren`